data/universal-pt.txt

```diff
@@ -0,0 +1,247 @@
+21 f R matthew : S. Mateus, Apóstolo e Evangelista
+23 m padre_pio : S. Pio de Pietrelcina, presbítero
+26 R cosmas_damian : S. Cosme e S. Damião, mártires
+27 m vincent_de_paul : S. Vicente de Paulo, presbítero
+28 R wenceslaus : S. Venceslau, mártir
+28 R ruiz : SS. Lourenço Ruiz e Companheiros, mártires
+29 f archangels : S. Miguel, S. Gabriel e S. Rafael, Arcanjos
+30 m jerome : S. Jerónimo, presbítero e doutor da Igreja
+
+= 10
+1 m therese_lisieux : S. Teresa do Menino Jesus, virgem e doutora da Igreja
+2 m guardian_angels : Santos Anjos da Guarda
+4 m francis_assisi : S. Francisco de Assis
+5 faustina_kowalska : Santa Faustina Kowalska
+6 bruno : S. Bruno, presbítero
+7 m bvm_rosary : Virgem Santa Maria do Rosário
+9 R denis : SS. Dionísio, bispo, e Companheiros, mártires
+9 leonardi : S. João Leonardo, presbítero
+11 m john_xxiii : S. João XXIII, papa
+14 R callistus_i : S. Calisto I, papa e mártir
+15 m teresa_avila : S. Teresa de Jesus, virgem e doutora da Igreja
+16 hedwig : S. Hedwiges, religiosa
+16 alacoque : S. Margarida Maria Alacoque, virgem
+17 m R ignatius_of_antioch : S. Inácio de Antioquia, bispo e mártir
+18 f R luke : S. Lucas, Evangelista
+19 R brebeuf_jogues : SS. João de Brébeuf e Isaac Jogues, presbíteros, e Companheiros, mártires
+19 paul_of_cross : S. Paulo da Cruz, presbítero
+22 m john_paul_ii : S. João Paulo II, papa
+23 john_capistrano : S. João de Capistrano, presbítero
+24 claret : S. António Maria Claret, bispo
+28 f R simon_jude : S. Simão e S. Judas, Apóstolos
+
+= 11
+1 s all_saints : TODOS OS SANTOS
+2 s V all_souls : Comemoração de Todos os Fiéis Defuntos 
+3 de_porres : S. Martinho de Porres, religioso
+4 m borromeo : S. Carlos Borromeu, bispo
+9 f2.5 lateran_basilica : Dedicação da Basílica de Latrão
+10 m leo_great : S. Leão Magno, papa e doutor da Igreja
+11 m martin : S. Martinho de Tours, bispo
+12 m R josaphat : S. Josafat, bispo e mártir
+15 albert : S. Alberto Magno, bispo e doutor da Igreja
+16 margaret_of_scotland : S. Margarida da Escócia
+16 gertrude : S. Gertrudes, virgem
+17 m elizabeth_of_hungary : S. Isabel da Hungria, religiosa 
+18 peter_paul_basilicas : Dedicação das Basílicas de S. Pedro e de S. Paulo, Apóstolos
+21 m bvm_presentation : Apresentação de Nossa Senhora
+22 m R cecilia : S. Cecília, virgem e mártir
+23 R clement_i : S. Clemente I, papa e mártir
+23 columban : S. Columbano, abade
+24 m R vietnamese_martyrs : SS. André Dung-Lac, presbítero, e Companheiros, mártires
+25 R catherine_of_alexandria : S. Catarina Alexandrina, virgem e mártir 
+30 f R andrew : S. André, Apóstolo
+
+= 12
+3 m francis_xavier : S. Francisco Xavier, presbítero, Padroeiro das Missões
+4 john_damascene : S. João Damasceno, presbítero e doutor da Igreja
+6 nicholas : S. Nicolau, bispo
+7 m ambrose : S. Ambrósio, bispo e doutor da Igreja
+8 s bvm_immaculate : IMACULADA CONCEIÇÃO DA VIRGEM SANTA MARIA, Padroeira principal de Portugal e das Dioceses de Évora, Santarém, Setúbal e Vila Real
```

Comment:
One more thing: the title `Padroeira principal de Portugal e das Dioceses de Évora, Santarém, Setúbal e Vila Real` does not belong here, as it is not part of the General Roman Calendar, but of the proper calendar of Portugal.

Reply:
makes sense, I still haven't created the proper for Portugal, but will move it there once I can

Reply:
> I still haven't created the proper for Portugal, but will move it there once I can

The policy of this project is not to accept contributed data files of particular (national, diocesan, local, religious etc.) calendars because it would be hard to keep all of them up-to-date. Contributions of such calendars are warmly accepted over at https://github.com/calendarium-romanum/data-contrib

For purposes of finalizing this PR please just delete the local patronal title. The `data/universal-XX.txt` are supposed to contain translations of the "Calendarium Romanum Generale" and the title therefore does not belong there.

Reply:
sure, I'll open the PRs with local files to the other repo then :shipit:  I also added on the comment of the file the extra sources that I used to retrieve the informations